manifests/init.pp

```diff
@@ -13,15 +13,19 @@
 class argus_server (
   $argus_host_dn,
   $admin_password,
+  $site_name,
```

Comment:
Please set a default value otherwise it will cause errors on existing installations.